app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,9 @@
   <body>
     <nav><%= link_to "Shelter Index", "/shelters", class: "nav-link" %> |
     <%= link_to "Pet Index", "/pets", class: "nav-link" %></nav>
+    <% flash.each do |type, message| %>
```

Comment:
should get the flash message at the top of each page now, regardless of the message